incubator/busbar/.helmignore

```diff
@@ -0,0 +1 @@
+.git
```

Comment:
The standard helmignore file has a bit more to it. For reference see https://github.com/kubernetes/charts/blob/5bff4b77c5c2cd40e244bfdba72b068248384745/stable/artifactory/.helmignore.